code/modules/food_and_drink/drinks.dm

```diff
@@ -290,12 +291,12 @@
 
 	proc/set_stuff(var/name, var/icon_state)
 		src.name = "crushed [name]"
-		if (icon_state == "cola" || "cola-blue")
+		if (icon_state == "cola-1" || icon_state == "cola-2")
```

Comment:
why is this if statement even here, switches don't need to be exhaustive

Reply:
this was not resolved.

Reply:
sorry; i don't really understand what to do here since i didn't code this specifically. i didnt have any idea at all that this was a spot that needed help until pali mentioned it.